### DISCUSSION TITLE
unable to save mesh ender3v2 4.2.2 10x10 bltouch 3.1

### DISCUSSION THREAD

**Discussion by KiLLie2k** [Jyers/Marlin#522] [Q&A] [answered]
I have tried 3 diff SD cards and formatted before inserting, it creates the mesh into ram and tells me to save it but when i click save the ender just beeps once and i've checked the SD cards no rom is being saved. this firmware is amazing btw, I haven't had the bltouch error out even once since I flashed it :D so i know im on the right track haha. had so many problems with the original firmware and bltouch apart from half my motherboard connections were loose on arrival *4 hours to figure that one out* sigh

**Comment by Cal-LF** [comment, marked as answer]
Jyres FW saves to eprom not SD card.

**Reply by KiLLie2k** [reply]
ty very much, nice upgrade from the stock firmware!  I am able to load the mesh and print right off now, thanks for imparting the knowledge :)

**Comment by G4GNX** [comment]
The values are saved to Eeprom as Cal-LF pointed out, so they can be used.
Don't forget to put either G29 or M420 S in your slicer start code, just after G28

**Reply by Tw0bit** [reply]
I understand G29 runs the auto level again, but M420 loads a saved mesh - is that right?

I am running a 10x10 UBL mesh, so I dont want to relevel the bed every print (I will on my larger critical ones, but the kids toys not so critical). Is M420 S the better command?

**Reply by Tw0bit** [reply]
I put those commands in; but it errors out - Appears to be on the AutoTilt.

Send: N7 G29 L0*73
Recv: Done.
Recv: ok
Send: N8 G29 A*123
Recv: Unified Bed Leveling System v1.01 active
Recv: ok
Send: N9 G29 J2*67
Recv: Tilting mesh point 1/4
Recv: 
Recv:  T:29.00 /0.00 B:34.00 /0.00 @:0 B@:0
Recv: echo:busy: processing
Recv:  T:29.00 /0.00 B:34.00 /0.00 @:0 B@:0
Recv: echo:busy: processing
Recv:  T:29.00 /0.00 B:34.00 /0.00 @:0 B@:0
Recv: echo:busy: processing
Recv:  T:29.00 /0.00 B:34.00 /0.00 @:0 B@:0
Recv: echo:busy: processing
Recv:  T:29.00 /0.00 B:34.00 /0.00 @:0 B@:0
Recv: //action:notification Probing Failed
Recv: Error:Probing Failed
Changing monitoring state from "Starting" to "Cancelling"
